### ISSUE TITLE
INS-the countrys subid should match the prime ministers subid

### ISSUE THREAD

**Issue by jonteboi0707** [TNODevs/TNO-Bug-Tracker#3960]
### HOI4 version

1.15.3

### TNO version

1.7.0

### Which major expansions do you NOT have?

I have all

### Describe your suggestion

When Johannes Leimena gets in charge Indonesia should turn into a Christian democracy.

### Justifications

The countrys subid should match Johannes's subid like with the other prime ministers.

### Screenshots

![20241225143530_1](https://github.com/user-attachments/assets/92fd4d21-915a-4a7b-8ce8-9609ab7a411a)
![20241225143534_1](https://github.com/user-attachments/assets/ddb0984f-81f1-41ab-9c48-c56290d6dcdb)

**Comment by Marok132** [comment]
@Stormeagle449 @Jack121Q Could it be fixed, please?

**Comment by Jack121Q** [comment]
I disagree, regime policy and the reality of governance matter more than the ideology of a singular person.
